e2etests/node_scan_rhcos_test.go

```diff
@@ -150,6 +151,9 @@ func TestGRPCGetRHCOSNodeVulnerabilities(t *testing.T) {
 	for name, c := range cases {
 		t.Run(name, func(t *testing.T) {
 			c := c
+			if !features.RHCOSNodeScanning.Enabled() {
+				t.Skip("This test requires ROX_RHCOS_NODE_SCANNING=true")
```

Comment:
Can we forcibly enable the feature flag in the UT and always test the code? Would it make more sense, given we can enable the flag in dev with environment variables?

Reply:
Yes that would be great! The thing is that I tried to do this already and failed to enable it only for the e2e tests. We would need to tell the CI to build and deploy with that FF enabled. Do you maybe know where we could do this?

Reply:
I left a comment. You can enable it on a per-unit test basis